app/src/main/java/com/kickstarter/viewmodels/usecases/GetShippingRulesUseCase.kt

```diff
@@ -170,18 +167,16 @@ class GetShippingRulesUseCase(
      * Rewards are only added if available, and the method ensures no duplicates or incorrect entries
      * by filtering and categorizing in a single pass.
      *
-     * Once filtered, the updated state is emitted via [emitCurrentState].
      *
      * @param allAvailableShippingRules map of location ID to shipping rule available for the project
      * @param rule the selected shipping rule (e.g., user's chosen location)
      * @param rewards the full list of rewards associated with the project
      */
-    private suspend fun filterRewardsByLocation(
```

Comment:
This method should **not** be suspending, state emission should happen once this method is completed